fast-list.js

```diff
@@ -258,6 +299,11 @@ FastList.prototype = {
       );
     }
```

Comment:
It looks as though it may be to address the scenario when we have fewer list-items than the `maxItemCount` and don't want to render more than we need to?